<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="2">
       <widget class="QPushButton" name="mc_choosebtn">
        <property name="text">
         <string>Choose Audio file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ae_inpath">
        <property name="placeholderText">
         <string>Input file path</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QCheckBox" name="mc_dupchecker">
        <property name="text">
         <string>Overwrite duplicate file (if present)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="AE_Submit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QTreeView" name="mc_directory_view"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="mc_outpath">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Output file path</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="mc_submitbtn">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="mc_inpath">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Input file path</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QTreeView" name="ac_directoryview"/>
      </item>
      <item row="10" column="1">
       <widget class="QCheckBox" name="ae_dupchecker">
        <property name="text">
         <string>Overwrite duplicate file (if present)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="ae_choosebtn">
        <property name="text">
         <string>Choose Video file</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ae_outpath">
        <property name="placeholderText">
         <string>Output file path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Audio Extractor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Audio/Music Cleaner</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QComboBox" name="ai_choice">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertPolicy::InsertAtBottom</enum>
        </property>
        <item>
         <property name="text">
          <string>Demucs ht (default)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actioncopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actioncopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/img_icons/resources/copy.png</normaloff>:/new/img_icons/resources/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/img_icons/resources/clipboard.png</normaloff>:/new/img_icons/resources/clipboard.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditUndo"/>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditRedo"/>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
